.github/workflows/tests.yaml

```diff
@@ -0,0 +1,82 @@
+name: Tests
+
+on: [ push ]
+
+env:
+    COMPOSER_ROOT_VERSION: "dev-master"
+
+jobs:
+    build-matrix:
+        runs-on: ubuntu-latest
+        steps:
+            -   uses: actions/checkout@v2
+            -   uses: shivammathur/setup-php@v2
+                with:
+                    php-version: 7.4
+                    coverage: none
+                env:
+                    COMPOSER_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+            -   run: composer install --no-progress --ansi
+            -   run: vendor/bin/monorepo-builder validate
+            -   id: set-packages
+                run: echo "::set-output name=packages::$(vendor/bin/monorepo-builder packages-json)"
+        outputs:
+            packages: ${{ steps.set-packages.outputs.packages }}
+
+    tests:
+        needs: build-matrix
+        runs-on: ubuntu-latest
+        strategy:
+            fail-fast: false
+            matrix:
+                php-version:
+                    - '7.3'
+                    - '7.4'
+                    - '8.0'
```

Comment:
We can do this in a future PR, but I think it's nice to build this dynamically, using:
https://github.com/LongRunning/LongRunning/blob/main/.github/workflows/tests.yaml#L21-L22
https://github.com/LongRunning/LongRunning/blob/main/.github/workflows/tests.yaml#L25
https://github.com/LongRunning/LongRunning/blob/main/.github/workflows/tests.yaml#L33

Reply:
I installed `vendor/bin/easy-ci` but it gave me a error  i have to look into this

Reply:
```bash
./vendor/bin/easy-ci php-versions-json
```
```output                                                                                                                  
 [ERROR] Class 'Symplify\PHPStanRules\Exception\ShouldNotHappenException' not found    
```
```json
{
    "name": "simple-bus/simple-bus",
    "description": "Simple CommandBus and EventBus implementation",
    "license": "MIT",
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/mihaileu/RabbitMqBundle.git"
        }
    ],
    "autoload-dev": {
        "psr-4": {
            "SimpleBus\\CI\\": "src"
        }
    },
    "require-dev": {
        "php": "^7.3 || ^8.0",
        "ext-json": "*",
        "symplify/easy-ci": "9.0.0-rc1",
        "symplify/monorepo-builder": "9.0.0-rc1"
    },
    "config": {
        "sort-packages": true
    },
    "minimum-stability": "dev",
    "prefer-stable": true,
    "require": {
        "symfony/console": "^4.4 || ^5.0"
    }
}
```